### PR TITLE
HAAR-701 Set Expired status in yellow and allow it to be deactivated

### PR DESCRIPTION
**Pull request by karenmillermoj**

### CODE REVIEW

File: integration-tests/mockApis/nomisusersandroles.js [hunk 1]
```diff
@@ -13,6 +13,7 @@ const replicateUser = (times) =>
       name: 'Brixton (HMP)',
     },
     dpsRoleCount: i,
+    status: i % 2 === 0 ? 'OPEN' : 'LOCKED',
   }))
```
Comment (karenmillermoj) [inline integration-tests/mockApis/nomisusersandroles.js:17]: Added this to make test data more meaningful

File: views/searchDpsUsers.njk [hunk 2]
```diff
@@ -211,7 +211,11 @@
                                 {{ govukTag({
                                   text: u.status,
                                   classes: "govuk-tag--pink"
-                                }) if not u.active }}
+                                 }) if not u.active and u.status != 'EXPIRED' }}
+                                {{ govukTag({
+                                  text: u.status,
+                                  classes: "govuk-tag--yellow"
```
Comment (karenmillermoj) [inline views/searchDpsUsers.njk:217]: Expired displayed in yellow

File: views/userDetails.njk [hunk 3]
```diff
@@ -96,21 +96,40 @@
...
+                  }) }}
+                  </td>
+                  {% if showEnableDisable %}
+                    <td class="govuk-table__cell govuk-table__cell--numeric govuk-!-width-one-quarter">
+                      {{ linkButton(csrfToken, {
+                        text: "Deactivate account",
+                        action: staffUrl + "/deactivate",
+                        id: "deactivate-button"
+                      })}}
+                    </td>
```
Comment (karenmillermoj) [inline views/userDetails.njk:114]: Expired now displayed in yellow and if have permissions, can deactivate/lock the user (An 'EXPIRED' user is ALWAYS unlocked)

File: views/userDetails.njk [hunk 4]
```diff
@@ -96,21 +96,40 @@
                   </td>
                 {% endif %}
               {% else %}
-                <td class="govuk-table__cell">
+                {% if staff.accountStatus == 'EXPIRED' %}
+                  <td class="govuk-table__cell">
+                  {{ govukTag({
+                    text: 'Inactive account - ' + staff.accountStatus,
+                    classes: "govuk-tag--yellow",
```
Comment (karenmillermoj) [inline views/userDetails.njk:103]: ?? Not sure if this should say Inactive at all?? The Jira ticket doesn't explicitly say it

**Review by brightonsbox — APPROVED** [review]
LGTM

**Pull request merged**